Dockerfile

```diff
@@ -51,6 +50,9 @@ ENV RAILS_SERVE_STATIC_FILES=${RAILS_SERVE_STATIC_FILES:-true}
 ARG PORT
 ENV PORT=${PORT:-3000}
 
+ARG NODE_ENV
+ENV NODE_ENV=${NODE_ENV:-production}
```

Comment:
Production mode :rocket:

Reply:
It's what tells npm to run in production - meaning React will run in production mode